source/guide_resilio.rst

```diff
@@ -0,0 +1,125 @@
+.. highlight:: console
+
+.. author:: Martin Porsch <https://martin.pors.ch/martin@pors.ch>
```

Comment:
Funny misunderstanding here ☺️ either URL _or_ Email. I’ll update the guide later to clarify this.